src/storm/solver/LpSolver.cpp

```diff
@@ -3,65 +3,154 @@
 
 #include "storm/storage/expressions/Expression.h"
 #include "storm/storage/expressions/ExpressionManager.h"
+#include "storm/utility/macros.h"
 
 namespace storm {
 namespace solver {
 
 template<typename ValueType>
-LpSolver<ValueType>::LpSolver()
+RawLpConstraint<ValueType>::RawLpConstraint(storm::expressions::RelationType relationType, ValueType const& rhs, uint64_t reservedSize)
+    : _relationType(relationType), _rhs(rhs) {
```

Comment:
Do we use this underscore notation for private members in other places?

Reply:
😇 I'll revert that. I was looking into the pros & cons of having this and accidentally pushed it here.